### PR TITLE
[Issue #15]Doc user-guide/simple-nginx.md translated

### PR DESCRIPTION
**Pull request by zjx-immersion**
Related to #16

### CODE REVIEW

**Comment by zjx-immersion** [comment]
I signed it

**Comment by resouer** [comment]
I will review this doc

File: docs/user-guide/simple-nginx.md [hunk 1]
```diff
@@ -1,56 +1,53 @@
...
+为了访问你的nginx初始页面,你还不得不保证通过外部IP的通信是被允许的。那么就要通过让防火墙允许80端口通信才可以做到
 
-### Next: Configuration files
+### 接下来: 配置文件
 
-Most people will eventually want to use declarative configuration files for creating/modifying their applications.  A [simplified introduction](/docs/user-guide/simple-yaml)
-is given in a different document.
\ No newline at end of file
+
+大多数人最终都会响使用声明式的配置文件来创建或修改他们的应用程序。另外一个文档给出了一个[简单介绍](/docs/user-guide/simple-yaml)。
```
Comment (resouer) [inline docs/user-guide/simple-nginx.md]: 鉴于大多数人愿意使用声明式的配置文件来创建或修改他们的应用程序，这里有另外一个文档给出了 ....

File: docs/user-guide/simple-nginx.md [hunk 2]
```diff
@@ -1,56 +1,53 @@
...
 ```shell
 kubectl stop rc my-nginx
 ```
 
-### Exposing your pods to the internet.
+### 将你的pod暴露给外网.
 
-On some platforms (for example Google Compute Engine) the kubectl command can integrate with your cloud provider to add a [public IP address](/docs/user-guide/services/#external-services) for the pods,
-to do this run:
+在一些平台上（例如Google Compute Engine），kubectl命令能够集成云端提供的API来给pod分配公有IP地址[公有IP地址](/docs/user-guide/services/#external-services)，可以通过以下命令来实现：
```
Comment (resouer) [inline docs/user-guide/simple-nginx.md]: 在特定的云平台上（例如Google Compute Engine）...

File: docs/user-guide/simple-nginx.md [hunk 3]
```diff
@@ -1,56 +1,53 @@
...
 
 ```shell
 kubectl expose rc my-nginx --port=80 --type=LoadBalancer
 ```
 
-This should print the service that has been created, and map an external IP address to the service. Where to find this external IP address will depend on the environment you run in.  For instance, for Google Compute Engine the external IP address is listed as part of the newly created service and can be retrieved by running
+这个命令会打印出被创建的service,以及映射到这些service的外部IP地址. 对外的IP地址根你实际运行环境有关。例如，对于Google Compute Engine的外部IP地址会被列为新创建的服务的一部分，可以通在运行时检索。
```
Comment (resouer) [inline docs/user-guide/simple-nginx.md]: ...你实际运行环境有关。以Google Compute Engine为例，这些外部IP地址作为创建的service的一部分，可以使用如下指令查看到。

**Comment by zjx-immersion** [comment]
Please review my fixes, Thanks @resouer

**Comment by resouer** [comment]
lgtm, thanks a lot!

**Comment by hurf** [comment]
@resouer  I would still recommend squash the commits before merging it.

**Comment by resouer** [comment]
@hurf Sorry, did not notice that mess ...

**Pull request merged**